src/components/Education.tsx

```diff
@@ -174,7 +175,7 @@ const Education = () => {
     // Delete an Education Card
     const handleDelete = async (id:number) => {
     
-        axios.delete('http://localhost:8081/education/' + id)
+        axios.delete(url + '/education/' + id)
```

Comment:
@ddinh please store url + "/education" to a variable and pass here to minimize code duplication